### ISSUE TITLE
refactor to avoid jQuery

### ISSUE THREAD

**Issue by danieljuhl** [dennybritz/neal-react#5]
As far as I can see, `jQuery` is loaded only to use `getScript()`. Would it be an idea to refactor to vanilla JS instead? I'm happy to give it a try, if you interested in a PR. I don't like loading jQuery just for that single method.

**Comment by dennybritz** [comment]
I'm definitely in favor of removing dependencies, but I think jQuery is also required by the Bootstrap Javascript, isn't it?

**Comment by danieljuhl** [comment]
Yes, but only if you actually use any of the JavaScript. Right now it would be possible to use markup components only, but neal would still require jQuery for eg. Google Analytics. So I would like to avoid using jQuery for getScript, and only load it, if the Bootstrap JS is actually used.

**Comment by danieljuhl** [comment]
At least thats how Bootstrap v3 is - but I don't think Bootstrap v4 has changed on that part.

**Comment by dennybritz** [comment]
That's a good point, I'm not sure which Bootstrap components use the Javascript, the only one I'm sure about is the modal dialog. Let's do it and remove the jQuery dependency. Would be great if you want to work on that :)

In that case we should also add something to the README that defines for which components the user needs to include jQuery.

**Comment by danieljuhl** [comment]
I just looked, at it is not as clear at the new bootstrap site, but lets remove jQuery and simple apply info in the docs for neal, about which comp. required jQuery (when we know for sure it requires it) - I'll look into Bootstrap v4 at the same time, to see which comp. are in play.

**Comment by LorbusChris** [comment]
+1
